pkgs/applications/misc/lutris/default.nix

```diff
@@ -107,7 +106,8 @@ let
 
     # ZDOOM
     soundfont-fluid bzip2 game-music-emu
-  ] ++ qt5Deps
+  ] ++ (if isNull steam then [] else [steam])
```

Comment:
Why not add a `withSteam ? true` and use `++ lib.optional withSteam steam`.

Reply:
Sounds like a much better approach, yes!